docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Use a custom domain to expose a service
+---
+
+This tutorial shows how to set up your custom domain and prepare a certificate to use the domain for exposing a service. The components used are Gardener [External DNS Management](https://gardener.cloud/docs/concepts/networking/dns-managment/#external-dns-management) and [Certificate Management](https://gardener.cloud/docs/concepts/networking/cert-managment/).
+
+To learn how to expose a service, go to [this](./apix-01-expose-service-apigateway.md) tutorial.
+
+## Prerequisites
+
+If you use a cluster not managed by Gardener, install the required components manually. Follow these steps:
+
+1. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+2. Download the [External DNS Management](https://github.com/gardener/external-dns-management) and [Certificate Management](https://github.com/gardener/cert-management) projects locally.
+
+3. Enable vertical Pod autoscaling on your cluster. For example, for Google Cloud Platform, run:
+
+   ```bash
+   gcloud beta container clusters update {PROJECT_NAME} --enable-vertical-pod-autoscaling
+   ```
+
+4. Install the `external-dns-management` component in your Namespace:
```

Comment:
What if I want to expose another workload, in another namespace, using different domain? In the open-source multiple domains POC the tutorial shows how to expose two workloads/two domains.
Here we just have one.
Should I install these components AGAIN (in anohter namespace)? I think it's not the case, they are singletons in the cluster. It must be explained. And the tutorial should at least mention (maybe at the end) which steps to repeat to expose another domain. I can figure it out, but our users may not be able to.